### PR TITLE
[#614] myplanet-activities-synced(connects #614)

### PR DESCRIPTION
**Pull request by yuviii**

### CODE REVIEW

**Comment by yuviii** [comment]
@lmmrssa @leomaxi @rrijal53  Data stored in `myplanet_activities` database after sync.

Screenshot 2019-03-10 11 11 58

File: app/src/main/java/org/ole/planet/myplanet/ui/sync/SyncActivity.java [hunk 1]
```diff
@@ -249,6 +244,7 @@ public void onSyncFailed(final String s) {
     public void onSyncComplete() {
         DialogUtils.showSnack(findViewById(android.R.id.content), "Sync Completed");
         progressDialog.dismiss();
+        editor.putLong("lastSynced", System.currentTimeMillis()).apply();
```
Comment (rrijal53) [inline app/src/main/java/org/ole/planet/myplanet/ui/sync/SyncActivity.java]: @yuviii  you dont need to save last synced time here , it is already saved in destory method of SyncManager.java as `LastSync`

**Comment by yuviii** [comment]
@rrijal53 please review the change.

File: app/src/main/java/org/ole/planet/myplanet/service/UploadManager.java [hunk 2]
```diff
@@ -46,6 +52,30 @@ public static UploadManager getInstance() {
 
     public UploadManager(Context context) {
         dbService = new DatabaseService(context);
+        this.context = context;
+    }
+
+    public void uploadActivities(SuccessListener listener) {
+        ApiInterface apiInterface = ApiClient.getClient().create(ApiInterface.class);
+        SharedPreferences pref = PreferenceManager.getDefaultSharedPreferences(context);
+        try {
```
Comment (rrijal53) [inline app/src/main/java/org/ole/planet/myplanet/service/UploadManager.java]: @yuviii  still not working 

Screen Shot 2019-03-10 at 18 05 08

The problem is you are using default SharedPreference here, but the time is saved in private SharedPreference . Please check constructor of SyncManager and test properly.

**Pull request merged**